packages/binding-modbus/README.md

```diff
@@ -3,62 +3,98 @@
 ## Overview
 
 W3C Web of Things (WoT) Protocol Binding for [Modbus](https://en.wikipedia.org/wiki/Modbus) TCP [RFC](https://tools.ietf.org/html/draft-dube-modbus-applproto-00).
-This package uses [modbus-serial](https://www.npmjs.com/package/modbus-serial) as a low level client for Modbus TCP.
+This package uses [modbus-serial](https://www.npmjs.com/package/modbus-serial) as a low-level client for Modbus TCP.
 W3C WoT Binding Template for Modbus can be found [here](https://w3c.github.io/wot-binding-templates/bindings/protocols/modbus/index.html).
 
 Current Maintainer(s): [@relu91](https://github.com/relu91) [@fillobotto](https://github.com/fillobotto)
 
-## Protocol specifier
+## Client Example
+
+You can use a code like the following to use the binding. This specific code is interacting with one of the Eclipse Thingweb Test Things at <https://github.com/eclipse-thingweb/test-things/tree/main/things/elevator>.
+
+```js
+// Protocols and Servient
+Servient = require("@node-wot/core").Servient;
+ModbusClientFactory = require("@node-wot/binding-modbus").ModbusClientFactory;
+
+// create Servient and add Modbus binding
+let servient = new Servient();
+servient.addClientFactory(new ModbusClientFactory());
+
+async function main() {
+    let td = {}; // see https://github.com/eclipse-thingweb/test-things/blob/main/things/elevator/modbus/js/modbus-elevator.td.json
```

Comment:
Why don't you want to include the TD ?

Reply:
It is changing at the moment due to the changes in this PR. Once this PR is merged and that TD is updated, I can add it inline. Another alternative would be to add a fetch method but that would imply adding the HTTP binding